### PR TITLE
[minor] Integration tests for birt and adhoc reports

### PR DESCRIPTION
**Pull request by unnati-solanki-git**
The following changes are part of this PR:
- Integration to BIRT Reports
- Integration to Adhoc Reports

### CODE REVIEW

**Review by whitfiea — COMMENTED** [review]
@unnati-solanki-git Can you add a comment to the original description to say these changes have been tested and the tests work?

**Comment by leo-miran** [comment]
@unnati-solanki-git I deployed this branch into project **_leotest_** in **_squadci's_** cluster and it looks like the tasks are a little bit out of order. Please take a look at this image
image

In my opiniom even the manage tasks are a little bit messed up. It looks like the [phases](https://github.com/ibm-mas/cli/tree/squadci/tekton/src/pipelines/taskdefs/fvt-manage) need to be reviewed as well.

File: tekton/src/pipelines/taskdefs/fvt-manage/phase4.yml.j2 [hunk 1]
```diff
@@ -135,3 +135,44 @@
     - fvt-manage-esig-basic
     - fvt-manage-eaudit-basic
     - fvt-manage-mif-kafka-setup
+
+# Manage FVT BIRT Reports
+- name: fvt-manage-birt-report
```
Comment (leo-miran) [inline tekton/src/pipelines/taskdefs/fvt-manage/phase4.yml.j2]: This task needs to be added to phase5 runafter ->https://github.com/ibm-mas/cli/blob/squadci/tekton/src/pipelines/taskdefs/fvt-manage/phase5.yml.j2#L17-L22

File: tekton/src/pipelines/taskdefs/fvt-manage/phase4.yml.j2 [hunk 2]
```diff
@@ -135,3 +135,44 @@
...
+    - fvt-manage-escalation-action
+    - fvt-manage-autoscript-adddeletewo
+    - fvt-manage-jms-inbound
+    - fvt-manage-esig-basic
+    - fvt-manage-eaudit-basic
+    - fvt-manage-mif-kafka-setup
+    - fvt-manage-communication-temp
+
+# Manage FVT Adhoc Reports
+- name: fvt-manage-adhoc-report
```
Comment (leo-miran) [inline tekton/src/pipelines/taskdefs/fvt-manage/phase4.yml.j2:147]: This task needs to be added to phase5 runafter ->https://github.com/ibm-mas/cli/blob/squadci/tekton/src/pipelines/taskdefs/fvt-manage/phase5.yml.j2#L17-L22

**Review by leo-miran — CHANGES_REQUESTED** [review]

File: tekton/src/pipelines/taskdefs/fvt-manage/phase4.yml.j2 (hunk 1 above)
Comment (unnati-solanki-git) [reply tekton/src/pipelines/taskdefs/fvt-manage/phase4.yml.j2]: I made the changes and kept adhoc and birt report tests before the last phase. Is that okay? Let me know if this does not look good. 

Thanks!

File: tekton/src/pipelines/taskdefs/fvt-manage/phase4.yml.j2 (hunk 2 above)
Comment (unnati-solanki-git) [reply tekton/src/pipelines/taskdefs/fvt-manage/phase4.yml.j2:147]: I made the changes and kept adhoc and birt report tests before the last phase. Is that okay? Let me know if this does not look good.

Thanks!

Comment (leo-miran) [reply tekton/src/pipelines/taskdefs/fvt-manage/phase4.yml.j2:147]: I have pushed additional changes and tasks now look better. Slack thread where this was discussed
https://ibm-watson-iot.slack.com/archives/C04UR4PPCRY/p1696339457555709

**Comment by leo-miran** [comment]
Here's evidence of a successful run of this branch
image

Cloud you please review @whitfiea ?

**Review by leo-miran — DISMISSED** [review]
We had a successful run of this branch in squadci fyre cluster
image

**Review by leo-miran — APPROVED** [review]

**Review by whitfiea — APPROVED** [review]

**Pull request merged**